sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/GeneralSQLExpression.java

```diff
@@ -22,20 +22,34 @@
 import org.apache.spark.annotation.Evolving;
 
 /**
- * The general SQL string corresponding to expression.
+ * The general V2 expression corresponding to V1 expression.
```

Comment:
Can we document this that this assumes that SQL is able to be parsed in the source that implements this expression?

Reply:
I have tested, the SQL syntax is different between databases.
So ,we try to use V2 expression to replace SQL string.

Reply:
I meant documenting that this API should be used in the source that can understand SQL in this Javadoc.